functions/java-jre.bash

```diff
@@ -18,7 +18,7 @@ java_webupd8_archive() {
 }
 
 java_zulu(){
-  cond_redirect systemctl stop openhab2.service
+  cond_redirect systemctl is-active --quiet openhab2 && systemctl stop openhab2
```

Comment:
Nice clean solution! 👍

Reply:
AFAIK, `systemctl stop` doesn't run the stop executable if the service isn't active so would this be necessary?

Reply:
I think you're right from the logics perspective but it spits a warning which is what I want to avoid for sake of a clean output.
Ok, re-added the .service but just because I'm too lazy to extinct it everywhere else.